test/box-luatest/downgrade_test.lua

```diff
@@ -783,3 +724,87 @@ g.test_downgrade_user_auth_history_and_last_modified = function(cg)
         check_NO_new_auth_fields()
     end)
 end
+
+-- Check that SQL CHECK constraints could be downgraded.
+g.test_downgrade_sql_tuple_check_contraints = function(cg)
+    cg.server:exec(function()
+        local helper = require('test.box-luatest.downgrade_helper')
+
+        local _func = box.space._func
+        local _ck = box.space._ck_constraint
+
+        box.execute([[CREATE TABLE t(i INT PRIMARY KEY, a INT);]])
+        box.execute([[ALTER TABLE t ADD CONSTRAINT c CHECK (a > 10);]])
+
+        local s = box.space._space.index[2]:get{'T'}
```

Comment:
Nit. May be it is better to use more descriptive `index.name`.

Reply:
Done.